docs/guides/advanced-cluster-new-sharding-schema.md

```diff
@@ -1,24 +1,28 @@
 ---
-page_title: "advanced_cluster - Migration to new sharding schema and leveraging Independent Shard Scaling"
+page_title: "Migration Guide: Advanced Cluster New Sharding Schema"
 ---
 
+# Migration Guide: Advanced Cluster New Sharding Schema
+
 **Objective**: Guide users to migrate their existing advanced_cluster configurations to use the new sharding schema which was introduced in version `1.18.0`. Additionally, a section is included to describe how Independent Shard Scaling can be used once the new sharding schema is adopted. Exiting sharding configurations will continue to work but will have deprecation messages if not using the new sharding schema.
 
-- [Overview of schema changes](#overview)
-- [Migrating existing advanced_cluster type SHARDED](#migration-sharded)
-- [Migrating existing advanced_cluster type GEOSHARDED](#migration-geosharded)
-- [Migrating existing advanced_cluster type REPLICASET](#migration-replicaset)
-- [Leveraging Independent Shard Scaling](#leveraging-iss)
+- [Migration Guide: Advanced Cluster New Sharding Schema](#migration-guide-advanced-cluster-new-sharding-schema)
+  - [Overview of schema changes](#overview)
+    - [Migrate existing advanced\_cluster type SHARDED](#migration-sharded)
```

Comment:
Here and in all instances in this part: add backticks for SHARDED and things like it. Do it throughout.

Also I would remove the word "existing" as it doesn't add value.